app/views/trainees/degrees/_non_uk_degree_form.html.erb

```diff
@@ -1,10 +1,12 @@
-<%= register_form_with model: [@trainee, @degree_form.degree], local: true do |f| %>
+<%= register_form_with model: [@trainee, @degree_form], local: true do |f| %>
   <%= f.hidden_field :locale_code, value: "non_uk" %>
   <%= f.govuk_error_summary %>
 
   <h1 class="govuk-heading-l">Degree details</h1>
 
   <%= render FormComponents::Autocomplete::View.new(
+    f,
+    attribute_name: :subject,
```

Comment:
I'm trying to think whether there's a way we can pass the `FormComponents::Autocomplete::View` exactly what is needs (the `raw_subject` and the `subject`) here rather than passing the whole form object around. Maybe this is simplest though 🤔

Reply:
yeah.. was thinking of that but the con is that you are then duplicating pulling out the value every time you use one of these autocomplete components. And I think it would be preferable to disallow someone from getting this wrong, if they passed in another value other than from the "raw" version of that attribute name then it wouldnt make sense.